package.json

```diff
@@ -55,6 +55,7 @@
         "@types/gapi.client.sheets": "4.0.20201030",
         "@types/google.picker": "0.0.39",
         "@types/lodash-es": "4.17.6",
+        "@types/mapbox__vector-tile": "^1.3.0",
```

Comment:
Please remember to remove the `^`..

Reply:
I'm so sorry... I fixed it.
[chore: fix versioning](https://github.com/reearth/reearth-web/pull/388/commits/0042c1b36eefdcbf667c2de2882e0bcc3c7f4a9b)